lib/notification_service/notification_service/cli/cli_parser.py

```diff
@@ -121,11 +122,113 @@ class GroupCommand(NamedTuple):
 
 CLICommand = Union[ActionCommand, GroupCommand]
 
-VERSION_COMMAND = ActionCommand("version",
-                                "Shows the version of Notification.",
-                                lazy_load_command("notification_service.cli.commands.version_command.version"),
-                                [],
-                                "Shows the version of Notification.")
+ARG_SERVER_URI = Arg(
+    ("-s", "--server-uri"),
+    help="The uri of notification server",
+    default=None,
+)
+
+ARG_NAMESPACE = Arg(
+    ("-n", "--namespace"),
+    help="Namespace of the event. If not set, all namespaces would be handled",
+    default=None,
+)
+
+ARG_KEY = Arg(
+    ("-k", "--key"),
+    help="Key of the event. If not set, all keys would be handled",
```

Comment:
When sending events, what will handle the keys?

Reply:
I'll remove the unnecessary comments.